config/locales/alias.en.yml

```diff
@@ -0,0 +1,7 @@
+en:
+  booth:
+    one: 'stand'
+    other: 'stands'
```

Comment:
Check out the Ruby on Rails string method [pluralize](https://api.rubyonrails.org/classes/String.html#method-i-pluralize). No need to have 2 translations just for pluralized strings.

Reply:
@hennevogel Alright! I got confused with rails pluralize and i18n pluralize. But I think using rails pluralize makes sense as we don't need a count to apply different translation for "stand".